web_clickjack_protection/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Just install
```

Comment:
I think this file is not necessary

Reply:
Added it as a simple explainer for those that might wonder if there is any prerequisites needed e.g widget property.

Reply:
Fixed as per OCA standard.